fdbserver/Status.actor.cpp

```diff
@@ -641,6 +641,13 @@ struct RolesInfo {
 			if (grvLatencyBands.size()) {
 				obj["grv_latency_bands"] = addLatencyBandInfo(grvLatencyBands);
 			}
+
+			TraceEventFields const& grvBatchMetrics = metrics.at("GRVBatchLatencyMetrics");
+			if(grvBatchMetrics.size()) {
+				JsonBuilderObject priorityStats;
+				priorityStats["batch"] = addLatencyStatistics(grvBatchMetrics);
+				obj["grv_batch_latency_statistics"] = priorityStats;
```

Comment:
This is putting the stats in a separate object (`grv_batch_latency_statistics`). Instead, I think you'll need to pull the `priorityStats` object out of the two if blocks and add to the same one in each. Then if there is any data in it, you can do `obj["grv_latency_statistics"] = priorityStats`.

Reply:
yes.. ok